### ISSUE TITLE
{'mcc': 0.0, 'tp': 0, 'tn': 29593, 'fp': 4207, 'fn': 0, 'F1_score': 0.0, 'eval_loss': 0.3490116110236382}

### ISSUE THREAD

**Issue by GabrielLin** [Liu-Hy/nlp-contrib-graph#10]
After I ran all the scripts in `traing_sent` folder and ran `sent_binary` , the output is

INFO:simpletransformers.classification.classification_model1:{'mcc': 0.0, 'tp': 0, 'tn': 29593, 'fp': 4207, 'fn': 0, 'F1_score': 0.0, 'eval_loss': 0.3490116110236382}

The output of `sent_binary.py` is 

INFO:simpletransformers.classification.classification_model1:{'mcc': 0.0, 'F1_score': 0.5, 'eval_loss': 6.7894658806706145}

Both of them with a warning
```
/home/XXXX/.conda/envs/py36liuhy/lib/python3.6/site-packages/sklearn/metrics/_classification.py:873: RuntimeWarning: invalid value encountered in double_scalars
  mcc = cov_ytyp / np.sqrt(cov_ytyt * cov_ypyp)
```

Are they normal? Or I need to train all other models before running tests (I do have enough space at this moment)?

By the way, line 13 should be removed before successfully to ran `sent_multi.py` in `test1` folder because there is no `bi_labels` column in that file.

Thanks.

**Comment by Liu-Hy** [comment]
These are the expected outputs. In the testing scripts,  we make predictions without trying to compare them with the annotation. This is what we actually did in the evaluation phase, since we didn't have access to the annotation. Moreover, the relation annotation for subtask 3 has not been released by the organizer (and probably won't). In all the testing scripts, the reported scores are based on false labels (all zero for binary classification, all 'O' for NER, etc.), and are completely meaningless. 
Sorry for the confusion. I will change the 'test' in folder names to 'predict', and make it clearer in the document.

You don't need to run all the training scripts before predicting. After you run a training script, the corresponding model is saved, so that it will be loaded in the script for predicting the same task.

Yes, line 13 should be removed. I have made the update.
